Block/System/Config/Form/DefaultDropOffPointInstabox.php

```diff
@@ -5,59 +5,35 @@
 namespace MyParcelNL\Magento\Block\System\Config\Form;
 
 use Magento\Backend\Block\Template\Context;
-use Magento\Config\Block\System\Config\Form\Field;
 use Magento\Framework\App\ObjectManager;
-use Magento\Framework\Data\Form\Element\AbstractElement;
-use Magento\Backend\Block\Widget\Button;
 use MyParcelNL\Magento\Helper\Data;
 use MyParcelNL\Sdk\src\Model\Carrier\CarrierFactory;
 use MyParcelNL\Sdk\src\Model\Carrier\CarrierInstabox;
 
-class DefaultDropOffPointInstabox extends Field
+class DefaultDropOffPointInstabox extends DefaultDropOffPoint
 {
-    /**
-     * Path to template file in theme.
-     *
-     * @var string
-     */
-    protected $_template = 'MyParcelNL_Magento::default_drop_off_point.phtml';
-
     /**
      * @var \MyParcelNL\Sdk\src\Model\Consignment\DropOffPoint
      */
     private $dropOffPoint;
 
+    /**
+     * @param  \Magento\Backend\Block\Template\Context $context
+     * @param  array                                   $data
+     *
+     * @throws \Exception
+     */
     public function __construct(Context $context, array $data = [])
     {
-        $objectManager = ObjectManager::getInstance();
-        $helper  = $objectManager->create(Data::class);
+        $objectManager      = ObjectManager::getInstance();
+        $helper             = $objectManager->create(Data::class);
```

Comment:
heb je die objectmanager en helper niet bij alle carrier specifieke dropoffpoints nodig? dan zou ik die gewoon ook in de parent class doen, kan je __construct hier weglaten